src/index.html

```diff
@@ -6,9 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="MOYO_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="list_item is is-active">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Smartphones</a>
+          </li>
+          <li class="list_item" data-qa="hover">
+            <a href="#" class="nav_link">Laptops & Computers</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Gadgets</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Tablets</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Photo</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--You will not foolish me :) Misha told every page need h1 for better searching and for screen readers. Thas why we cant delete it or use "hidden" artribute or "visibility:hidden" prorerty-->
```

Comment:
good job)

Reply:
That actually was answering. Denys asked why I left H1 and then hid it. 
But in case You dont see these comments (cos I deleted pull request) this my answering looks studied so I deleted them 😂

Reply:
Mr mentor Andriy! 
Yesterday while I was fixing mistakes everything went wrong way and my code completely broke. 
Now I trying to  put everything together but it hard cos I got lost in Your comments. For example, I see two your comments which relate one of my mistake and there are  both "not fixed" and I cant find previous comment what exactly I need to fix here.  /* That's a pity there are no emojis, so --crying emoji --*/

So can I ask for NEW comments but not repeating old one? And can You also include selector from my code which You comment? I know there are numbering rows but when I, for example, delated all my "visually-hidden"  things which takes 9 rows and changed it by code You offered (it takes 2 rows), numbering system changed a lot... 

For now, I've changed everything I found in the comments except for one thing - line-height: 60px. I cant avoid this.
By using just "align-items: center" for "nav_item" without "line-height: 60px" appendix attacking word "Apple". Then I added padding-bottom for "Apple" and its center was shift .


P.S. Im not sure I add commit successfully. Somewhere it show 5 commit, in  other 7 commits. 
Maybe it cos I turned Draft mode. Instruction tells I can turn it off by clicking Ready for review but I dont see this option. 
And also maybe it cos I tried to restore commit to my workspace and something went wrong and then I cant make commit, so I used -force flag. 
OMG I have so many problems with my interest to pressing unknown buttons... I`ll try not to do that in the future.